pkg/reporter/namespace_report.go

```diff
@@ -68,6 +68,10 @@ type NamespaceReportHandler struct {
 	location voyager.Location
 }
 
+type ProviderOpsResponse struct {
+	Result *ProviderResponse `json:"report,omitempty"`
```

Comment:
Field name and json tag do not match. They probably should, at least to follow the convention that we have.